.travis.yml

```diff
@@ -19,7 +19,7 @@ install:
   # Carthage
   - carthage bootstrap --platform iOS
   # Update fastlane
-  - gem install fastlane -v '2.100.1'
+  - gem update fastlane
```

Comment:
Not a good thing. I had this once. The CI can brake quite easily.

Reply:
My idea was to avoid making a PR like this multiple times a year :)

But you are right, consistency is better.